### ISSUE TITLE
[Feature]: Auto-convert old YAML model config to JSON

### ISSUE THREAD

**Issue by brknsoul** [vladmandic/automatic#3350]
### Issue Description

Using [YiffyMix v42](https://civitai.com/models/3671?modelVersionId=405032) with the yaml file installed along side, model prediction type isn't being changed, resulted in burned images.

### Version Platform Description

```
Python version=3.10.11 platform=Windows bin="E:\sdmaster\venv\Scripts\python.exe"
venv="E:\sdmaster\venv"
Version: app=sd.next updated=2024-07-10 hash=2ec6e9ee branch=master
url=https://github.com/vladmandic/automatic/tree/master ui=main
Latest published version: a874b27e50a343ac55148756a0a80eafb3a7f87f 2024-07-24T20:16:33Z
Platform: arch=AMD64 cpu=Intel64 Family 6 Model 151 Stepping 5, GenuineIntel system=Windows
release=Windows-10-10.0.19045-SP0 python=3.10.11
```

### Backend

Diffusers

### UI

ModernUI

### Branch

Master

### Model

StableDiffusion 1.5

### Acknowledgements

- [X] I have read the above and searched for existing issues
- [X] I confirm that this is classified correctly and its not an extension issue

**Comment by vladmandic** [comment]
diffusers no long use yaml config files, they use json configuration instead.

If only thing needed is to switch to v-prediction mode, you can do that in settings.

**Comment by brknsoul** [comment]
> If only thing needed is to switch to v-prediction mode, you can do that in settings.

yes, but most models use default. The yaml file is mean to auto-switch to v-predication for models that request it.

**Comment by vladmandic** [comment]
True - and thats why I didn't close the issue, only noted the background.

**Comment by bmhRi3kXAFSw** [comment]
Is there any tool in the UI itself to convert configs to json format? There is a python script floating around for converting safetensors to diffusers format, but I've never gotten it to work and load correctly.

**Comment by vladmandic** [comment]
not as far as I know, but shouldn't be too hard to write config converter - that's why I'm keeping this issue open - I'll take a look when I get back from my trip.

**Comment by vladmandic** [comment]
i was monitoring if more models are going to pop-up needing this, but half a year later and this single model is the only reported instance so far so i don't think its worth doing auto-convert config yaml to json just to fit a single legacy model, especially since its not needed for sdxl variant, only for sd15 variant.
